pkgs/by-name/wi/wifitui/package.nix

```
@@ -0,0 +1,24 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub
+}: buildGoModule rec {
+  pname = "wifitui";
+  version = "unstable";
+
+  src = fetchFromGitHub {
+    owner = "greedoftheendless";
```

I think i would be better to use the mainstream repository instead of your fork
```suggestion
    owner = "shazow";
```